src/main/java/com/snowflake/kafka/connector/internal/streaming/TopicPartitionChannel.java

```diff
@@ -38,11 +36,14 @@
  */
 public class TopicPartitionChannel {
   private static final Logger LOGGER = LoggerFactory.getLogger(TopicPartitionChannel.class);
-  private static final ObjectMapper MAPPER = new ObjectMapper();
 
   // more of previousInsertRowTsMs
   private long previousFlushTimeStampMs;
 
+  /**
+   * States whether this channel has had any data before. If this is false, the
+   * topicPartitionChannel has recently been initialised and didnt receive any records before
```

Comment:
nit:  The comment is a little confusing since the variable `hasChannelInitialized ` is false if the channel has recently been initialized

Reply:
hmm, may be I can rename it to something more descriptive.. 
`hasChannelReceivedAnyRecordsBefore`